File: 1-student_details_entry_SpringBoot/src/main/java/com/daram/app1/entity/Employee.java
package com.daram.app1.entity;

import java.sql.Blob;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Lob;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@NoArgsConstructor
public class Employee {

	@Id
	@GeneratedValue(strategy = GenerationType.AUTO)
	private Integer empId;
	private String name;
	private String email;
	private Integer age;
	private String gender;
	private Integer mobile;
	private  String state;
	private String password;
	@Lob
	@Column(columnDefinition = "BLOB")
	private byte[] profilePhoto;
	private Integer activeStatus;    //0 inactive, 1 active
	private String verificationCode;
	
	
}

File: 1-student_details_entry_agular/src/app/model/Employee.ts
export class Employee{
    name: string;
    email: string;
    age: Number;
    gender: string;
    mobile: Number;
    state: string;
    password: string;
    // constructor(name: string, email: string, age: Number, gender: string, mobile: Number, state: string, password: string)
    // {
    //     this.name = name;
    //     this.email = email;
    //     this.age = age;
    //     this.gender = gender;
    //     this.mobile = mobile;
    //     this.state = state;
    //     this.password = password;
    // }
    constructor()
    {
        this.name = '';
        this.email = '';
        this.age = 0;
        this.gender = ''
        this.mobile = 0;
        this.state = '';
        this.password = '';
    }
}